### DISCUSSION TITLE
unsupported JPEG feature: unknown marker

### DISCUSSION THREAD

**Discussion by monarc83** [photoprism/photoprism#2039] [Q&A] [answered]
I have currently four photos listed under hidden with a broken thumbnail and the following error messages in the details.

`unsupported JPEG feature: unknown marker`
`invalid JPEG format: unknown marker`
`invalid JPEG format: bad Pq value`

Any idea where this comes from and how to fix it?

Any help is much appreciated. Thanks in advance.

**Comment by graciousgrey** [comment, marked as answer]
Maybe those JPEGs are broken or not standard conform. Can you open those files with other tools?

**Reply by lastzero** [reply]
https://github.com/photoprism/photoprism/issues/1673

**Reply by monarc83** [reply]
> Maybe those JPEGs are broken or not standard conform. Can you open those files with other tools?

Yes, I can open them without any problem with other tools.

**Reply by lastzero** [reply]
Yes, some tools ignore errors - the implementation in Go (provided by Google) does not.

**Reply by lastzero** [reply]
see https://go-review.googlesource.com/c/go/+/382754/

**Reply by monarc83** [reply]
okay, so there is nothing I can do about it?

**Reply by lastzero** [reply]
Well, you could use a tool that fixes the glitch if needed. Or wait until Google had time to review and merge the pull request. Feel free to tell them how important this is :)
